<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NachtlagerDialog</class>
 <widget class="QDialog" name="NachtlagerDialog">
  <property name="windowTitle">
   <string>Nachtlager bearbeiten</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="labelOrt">
       <property name="text">
        <string>Ort</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEditOrt"/>
     </item>

     <item row="1" column="0">
      <widget class="QLabel" name="labelAbfahrt">
       <property name="text">
        <string>Abfahrt</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEditAbfahrt"/>
     </item>

     <item row="2" column="0">
      <widget class="QLabel" name="labelAnkunft">
       <property name="text">
        <string>Ankunft</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEditAnkunft"/>
     </item>

     <item row="3" column="0">
      <widget class="QLabel" name="labelPreis">
       <property name="text">
        <string>Preis</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBoxPreis">
       <property name="maximum">
        <double>10000.0</double>
       </property>
       <property name="decimals">
        <number>2</number>
       </property>
      </widget>
     </item>

     <item row="4" column="0">
      <widget class="QLabel" name="labelUnterkunft">
       <property name="text">
        <string>Unterkunfts-Typ</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="comboBoxUnterkunft">
       <item><property name="text"><string>Bergplateau</string></property></item>
       <item><property name="text"><string>HÃ¶hle</string></property></item>
       <item><property name="text"><string>Baumkrone</string></property></item>
      </widget>
     </item>

     <item row="5" column="0">
      <widget class="QLabel" name="labelRuhelevel">
       <property name="text">
        <string>Ruhelevel</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBoxRuhelevel">
       <property name="minimum">
        <double>0.0</double>
       </property>
       <property name="maximum">
        <double>10.0</double>
       </property>
       <property name="decimals">
        <number>1</number>
       </property>
       <property name="singleStep">
        <double>0.1</double>
       </property>
      </widget>
     </item>

    </layout>
   </item>

   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>

  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
